tests/utests/types/int8.c

```diff
@@ -0,0 +1,1720 @@
+/**
+ * @file int8.c
+ * @author Radek Iša <isa@cesnet.cz>
+ * @brief test for int8 values
+ *
+ * Copyright (c) 2020 CESNET, z.s.p.o.
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#define _POSIX_C_SOURCE 200809L /* strdup */
+#include <ctype.h>
+
+#define  _UTEST_MAIN_
+#include "../utests.h"
```

Comment:
this one include with the `_UTEST_MAIN_` is an exception to the order of includes, it is supposed to be always the first.
I see a problem in the `utests.h` regarding the ordering, but I'm going to fix it in standalone bugfix, the rule is that `utests.h` is included first in all tests